app/src/components/ReviewCard.tsx

```diff
@@ -0,0 +1,57 @@
+import { IconButton } from "@material-ui/core"
+import CancelIcon from "@material-ui/icons/Cancel"
+import CheckCircleIcon from "@material-ui/icons/CheckCircle"
+import BowlCard from "components/BowlCard"
+import PlayerChip from "components/PlayerChip"
+import { CurrentGameContext } from "contexts/CurrentGame"
+import {
+  CurrentGameSubscription,
+  useAcceptCardMutation,
+  useRejectCardMutation,
+} from "generated/graphql"
+import * as React from "react"
+
+function ScreenCard(props: {
+  card: CurrentGameSubscription["games"][0]["cards"][0]
+}) {
+  const currentGame = React.useContext(CurrentGameContext)
+  const [acceptCard] = useAcceptCardMutation()
+  const [rejectCard] = useRejectCardMutation()
+
+  const player = currentGame.players.find(
+    (player) => player.id === props.card.player_id
+  )
+
+  return (
+    <BowlCard>
+      <PlayerChip username={player?.username || ""}></PlayerChip>
+      <div>{props.card.word}</div>
+      <div>
+        <IconButton
+          onClick={() => {
+            rejectCard({
+              variables: {
+                id: props.card.id,
+              },
+            })
+          }}
+        >
+          <CancelIcon></CancelIcon>
+        </IconButton>
+        <IconButton
+          onClick={() => {
+            acceptCard({
+              variables: {
+                id: props.card.id,
+              },
+            })
+          }}
+        >
+          <CheckCircleIcon></CheckCircleIcon>
+        </IconButton>
+      </div>
+    </BowlCard>
+  )
+}
+
+export default ScreenCard
```

Comment:
todo: styling

Reply:
Here's where this landed in 56b6bcf1de3a69f52bb27bad6440837a5257d51b:

<img width="557" alt="Screen Shot 2020-07-15 at 9 10 30 PM" src="https://user-images.githubusercontent.com/847532/87615046-b41c9480-c6df-11ea-94bb-4c1202f47487.png">

I couldn't easily figure out how to create green buttons so just rolled with this for now 🤷‍♂️

Reply:
I did something like this- 

![image](https://user-images.githubusercontent.com/2534903/87626133-82f09400-c6e0-11ea-9182-22a10218328a.png)

Thoughts?

Reply:
Works for me, thanks!